fedmsg/commands/__init__.py

```diff
@@ -81,6 +81,9 @@ def _daemonize(self):
         except:
             from daemon.pidlockfile import PIDLockFile
 
+        msg = ('The use of the "--daemon" flag is deprecated and will be removed in '
+               'fedmsg-0.20.0. Use your init system to run fedmsg commands as daemons.')
+        warnings.warn(msg, category=UserWarning)
```

Comment:
I guess this code isn't covered by tests, which is why this is causing coverage to go down. I would recommend writing a test to hit this and maybe make some assertions about the method.